.travis.yml

```diff
@@ -35,4 +30,4 @@ script:
    if [ $GHCVER = "head" ] || [ ${GHCVER%.*} = "7.8" ] || [ ${GHCVER%.*} = "7.6" ] || [ ${GHCVER%.*} = "7.4" ] ; then
       cabal install --constraint="optparse-applicative < $VERSION"
       optparse-applicative-tests
-   fi
\ No newline at end of file
+   fi
```

Comment:
Sadly wouldn't work yet for GHC 8.0, as `tasty` has no install-plan for GHC 8 yet....